docs/addons/teams.md

```diff
@@ -0,0 +1,33 @@
+---
+id: teams
+title: Microsoft Teams
+---
+
+> This feature was introduced in \_Unleash v3.11.x.
+
+The MicrosoftTeams addon allows Unleash to post Updates when a feature toggle is updated. To set up this addon, you need to set up a webhook connector for your channel. You can follow [Creating an Incoming Webhook for a channel](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook) on how to do that.
+
+The Microsoft Teams addon will perform a single retry if the HTTP POST against the Microsoft Teams Webhook URL fails (either a 50x or network error). Duplicate events may happen, and you should never assume events always comes in order.
+
+## Configuration
+
+#### Events
+
+You can choose to trigger updates for the following events (we might add more event types in the future):
+
+- feature-created
+- feature-updated
+- feature-archived
+- feature-revived
+- feature-stale-on
+- feature-stale-off
```

Comment:
I wish there was an event for a `feature-requested-that-doesnt-exist` ;) Maybe I'll add that...

Reply:
I think you can solve that by creating a new feature toggle with a proper description? Then the software team should just use that toggle and start creating the new feature :dancers:

Reply:
True. The only hiccup we have is that we are running 3 instances of Unleash (dev, non-prod, prod) So it is possible that an upper lane would start asking for a toggle that hasn't been created (we are still finding our preferred process)

I think you are correct, though.